sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
 # Release History
 
-## 11.0.0-beta.2 (2020-04-03)
+## 1.0.0-beta.2 (2020-04-03)
 
+- Renamed the azure-search module to azure-search-documents.
+- Changed the namespace com.azure.search to com.azure.search.documents.
 - Added support for continuation tokens to resume server-side paging.
 - Replaced `SearchApiKeyCredential` with `AzureKeyCredential`.
+- Moved `AzureKeyCredentialPolicy` to Azure Core.
 - Fixed a bug where the Date header wouldn't be updated with a new value on request retry.
+- Changed the field type of `CustomAnalyzer`.
+- Made `RangeFacetResult` and `ValueFacetResult` object strongly typed.
+- Added helper function for IndexBatchException. 
+- Added ScoringParameter class.
+- Refactored some boolean field getter.
+- Made `IndexDocumentsBatch` APIs plurality.
 
 ## 11.0.0-beta.1 (2020-03-10)
```

Comment:
@alzimmermsft what should happen here, with the versions in the changelog, since this library has been renamed? Also, if this is being re-versioned and the previous release was only a beta should we even keep the 11* version stuff in here?

Reply:
Synced offline. Leave the change log as it is so that customer is aware of the version history.

Reply:
Added notes to make it clear.